apps/test/unit/lib/kits/maker/ui/MakerSetupGuideTest.js

```diff
@@ -0,0 +1,66 @@
+/** @file Test SetupGuide component */
+import React from 'react';
+import {expect} from '../../../../../util/deprecatedChai';
+import {shallow} from 'enzyme';
+import MakerSetupGuide from '@cdo/apps/lib/kits/maker/ui/MakerSetupGuide';
+import applabI18n from '@cdo/applab/locale';
+import experiments from '@cdo/apps/util/experiments';
+
+describe('MakerSetupGuide', () => {
+  describe('Microbit experiment is not enabled', () => {
+    let description;
```

Comment:
Only really useful to define stuff out here if they are being set in a `before` or `beforeEach` so the same thing is seen by all tests. Since you don't do that... there is no need to place these here. They can be defined inside the `it` where they are first assigned and used.